### DISCUSSION TITLE
make the Info Language option present in general tab present  in a show settings an option selectable when we add a new show sot it can be set by default for all future show

### DISCUSSION THREAD

**Discussion by tristelune** [pymedusa/Medusa#11724] [Ideas & Feature requests]
hello

i have set my Default Indexer Language to french in general tab but each time i add a show despite the fact that on the search page : i select french and the mention : * **This will only affect the language of the retrieved metadata file contents and episode filenames.**
This DOES NOT allow Medusa to download non-english TV episodes! 

all my show metadata are still downloaded in English until i go the show settings and change the option : Info Language to french and make a full re scan i would love to have on the add new show page Step 3 Customize options the setting : Info Language to be able to change it **and set it as default option for all show.**

It would be much more convenient or make Info Language option to default to the Default Indexer selection it would make much more sense, but maybe harder to update.
I tend to always forget about this and keep wondering why it stop working until i found it back again on the web then i have to update a lots of show manually each time to correct this.

Many Thank to consider this for a future update.

**Comment by Mika3578** [comment]
fixed in https://github.com/pymedusa/Medusa/pull/12202
